.github/workflows/ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           make -C build/ all
 
       - name: Integration Tests
-        run:  ctest --test-dir build --output-on-failure | tee -a $GITHUB_STEP_SUMMARY
```

Comment:
Could you add the "| tee -a $GITHUB_STEP_SUMMARY" back to each of the unit test runs?

Reply:
The tee masks the exit code of the previous command. This means that the tests can actually fail, but the exit code of the tee is read marking it as a success.
Link to a run where this happened to show what I'm talking about can be seen [here](https://github.com/Skptak/corePKCS11/actions/runs/6896046565/job/18761331009#step:4:1)
```
-----------------------
30 Tests 1 Failures 0 Ignored 
FAIL


67% tests passed, 1 tests failed out of 3

Total Test time (real) =   0.03 sec

The following tests FAILED:
	  3 - pkcs11_wrapper_utest (Failed)
```